### ISSUE TITLE
Displayed consent text is hardcoded

### ISSUE THREAD

**Issue by Werkov** [fykosak/dokuwiki-plugin-cookielaw#2]
The displayed consent text is hardcoded as localized Dokuwiki string. However, it would be better to have it more user configurable (and not GA specific only).

**Comment by tangotribe** [comment]
I agree! Default text seems unfriendly to me. I want it as a Configuration Setting.
